packages/styles/scss/patterns/FeaturedLink/_featuredlink-item.scss

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin featuredlink-item {
+  .#{$prefix}--featuredlink-item {
+    padding-top: $carbon--layout-01;
+    padding-left: $carbon--layout-01;
+    background: $ui-interactive;
+    display: flex;
+    flex-direction: column;
+    flex: 100%;
+    height: 100%;
+    @include carbon--breakpoint-down('lg') {
+      flex: 0 50%;
+    }
+    &__title {
+      margin-bottom: $carbon--layout-03;
+      width: 90%;
+      max-width: 640px;
+      color: #ffffff;
```

Comment:
you can use the `$text-04` color token here

https://www.carbondesignsystem.com/guidelines/themes

Reply:
@annawen1 resolved .. and i made it to $text-04